src/client/components/MyArea/MyArea.tsx

```diff
@@ -70,14 +68,13 @@ export interface MyAreaProps {
 function updateViewportHeight() {
   document.documentElement.style.setProperty(
     '--map-container-height',
-    `${window.innerHeight}px`
+    `${window.innerHeight - 150}px`
```

Comment:
150: magic number? Kun je die in een constant zetten met naam/beschrijving erbij?